charts/topolvm/values.yaml

```diff
@@ -735,3 +735,11 @@ priorityClass:
 snapshot:
   # snapshot.enabled -- Turn on the snapshot feature.
   enabled: true
+
+crds:
```

Comment:
When editing `values.yaml` file, we should update `charts/topolvm/README.md` by `make generate-helm-docs`.

```diff
$ make generate-helm-docs
./bin/helm-docs -c charts/topolvm/
INFO[2024-10-04T08:58:00+09:00] Found Chart directories [., charts/cert-manager]
INFO[2024-10-04T08:58:00+09:00] Generating README Documentation for chart charts/topolvm/charts/cert-manager
INFO[2024-10-04T08:58:00+09:00] Generating README Documentation for chart charts/topolvm
$ git diff
diff --git a/charts/topolvm/README.md b/charts/topolvm/README.md
index d5e5beaee448..7435e442ccdc 100644
--- a/charts/topolvm/README.md
+++ b/charts/topolvm/README.md
@@ -40,6 +40,7 @@ See [Getting Started](https://github.com/topolvm/topolvm/blob/topolvm-chart-v15.
 | controller.tolerations | list | `[]` | Specify tolerations. # ref: https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/ |
 | controller.updateStrategy | object | `{}` | Specify updateStrategy. |
 | controller.volumes | list | `[{"emptyDir":{},"name":"socket-dir"}]` | Specify volumes. |
+| crds.keep | bool | `true` |  |
 | env.csi_provisioner | list | `[]` | Specify environment variables for csi_provisioner container. |
 | env.csi_registrar | list | `[]` | Specify environment variables for csi_registrar container. |
 | env.csi_resizer | list | `[]` | Specify environment variables for csi_resizer container. |
$ 
```